cmd/kubermatic-installer/cmd_deploy.go

```diff
@@ -120,6 +123,7 @@ func DeployCommand(logger *logrus.Logger, versions kubermaticversion.Versions) *
 	cmd.PersistentFlags().DurationVar(&opt.HelmTimeout, "helm-timeout", opt.HelmTimeout, "time to wait for Helm operations to finish")
 	cmd.PersistentFlags().StringVar(&opt.HelmBinary, "helm-binary", opt.HelmBinary, "full path to the Helm 3 binary to use")
 	cmd.PersistentFlags().BoolVar(&opt.SkipDependencies, "skip-dependencies", false, "skip pulling Helm chart dependencies (requires chart dependencies to be already downloaded)")
+	cmd.PersistentFlags().Var(flagopts.SetFlag(opt.SkipSeedValidation), "skip-seed-validation", "comma-separated list of seed clusters to skip running the preflight checks on")
```

Comment:
I now it's a nitpick, but can we make the message a bit more explicit that this can result in broken seeds/user clusters? It doesn't really highlight that this can be somewhat dangerous.

Reply:
If that makes you accept this PR morally, then absolutely we can!

Reply:
I would sleep slightly better at night!